presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/NoisyCountAndSumAggregationUtils.java

```diff
@@ -22,13 +22,13 @@
 import static com.facebook.presto.common.type.DoubleType.DOUBLE;
 import static com.facebook.presto.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
 
-public class NoisySumAggregationUtils
+public class NoisyCountAndSumAggregationUtils
```

Comment:
nit: perhaps just call it NoisyAggregationUtils

Reply:
There is also another Util file for NoisyCount that does things specifically to Count and (in my next PR for noisy count_if) will have similar function names with the functions in this file. So I think it is better to keep them in separate files